src/writeFile.js

```diff
@@ -31,37 +31,20 @@ export default function writeFile(globalRef, pattern, file) {
             if (pattern.toType === 'template') {
```

Comment:
Please split into separate PR, bc of the comment above

Reply:
@michael-ciniawsky after upgrade this code do error, need remove this lines, that everything worked as it should

Reply:
So the dependency upgrade is required for these changes ? What exactly errors atm (I'm not too familiar with this package yet tbh 😅 )

Reply:
@michael-ciniawsky yes with .dot files and files without directory `foo.txt`

Reply:
#137 ? If see that one, but the issue originates from `loaderUtils.interpolateName()` ?

Reply:
@michael-ciniawsky this problems avoid after upgrade